news/2023/2023-12-14-osu-mania-7k-world-cup-2024-registrations-now-open.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2024: Registrations Now Open!"
+date: 2023-12-14 15:00:00 +0000
+---
+
+Winter season has arrived, and so has the first 2024 osu! World Cup, but this time with a twist!
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+After witnessing the amazing close to the 2023 tournament season with the osu! World Cup (for which we have an awesome recap that you can read [here](https://osu.ppy.sh/home/news/2023-12-10-osu-world-cup-2023-concludes), if you didn't already), it's time to kick off 2024 with key smashing of the 7 key variety.
+
+Seeing the amazing player turnout for last year's 7K MWC, we have decided to expand the tournament format to have **24 teams** going out of the qualifier stage instead of the usual 16. This means that the tournament will have one extra week (which also makes it have the same duration as the other World Cups), and that the top 8 teams from the bracket get one week off after the qualifiers in the form of a bye.
```

Comment:
more variety compared to previous paragraph

```suggestion
Last year's turnout for 7K MWC was huge. So much so, that we have decided to expand the tournament format to have **24 teams** going out of the Qualifier stage instead of the usual 16. This means the tournament will have one extra week (making it the same duration as the other World Cups), and the top 8 teams from the bracket get one week off after the Qualifiers in the form of a bye.
```